### ISSUE TITLE
LSCG keeps resetting

### ISSUE THREAD

**Issue by Blitzerfnv** [littlesera/LSCG#375]
I have no clue why it does that. I load from BCX btw, and all the settings reset every time.

**Comment by littlesera** [comment]
Do you mean FBC? I don't think you can load LSCG via BCX..?

**Comment by Blitzerfnv** [comment]
Yeah FBC, and what Player.LSCG and Player.OnlineSettings.LSCG? Where are those?

**Comment by littlesera** [comment]
They are objects you can view in the dev tools console
F12 or Ctrl+Shift+I in chrome.

**Comment by littlesera** [comment]
![image](https://github.com/littlesera/LSCG/assets/126942824/b3d16192-df31-47c7-bbd4-e7ae3a5bf118)
